.github/workflows/continuous-integration.yml

```diff
@@ -131,5 +131,6 @@ jobs:
         sed -i "s/smithy-kotlin-runtime-version = .*$/smithy-kotlin-runtime-version = \"$SMITHY_KOTLIN_RUNTIME_VERSION\"/" gradle/libs.versions.toml
         sed -i "s/smithy-kotlin-codegen-version = .*$/smithy-kotlin-codegen-version = \"$SMITHY_KOTLIN_CODEGEN_VERSION\"/" gradle/libs.versions.toml
         ./gradlew --parallel publishToMavenLocal
+        ./gradlew build
```

Comment:
What is the reason for adding `./gradlew build` here? Don't the `test` tasks depend on `build` already?

Reply:
I think so, the `build` task depends on the `stageServices` task but it doesn't seem to be running when the `build` task is being called by the `test` task.

I added a dependency on `stageServices` to the `test` tasks and that should allow us to get rid of this ^